src/libstd/keyword_docs.rs

```diff
@@ -608,6 +608,63 @@ mod in_keyword { }
 /// [Reference]: ../reference/statements.html#let-statements
 mod let_keyword { }
 
+#[doc(keyword = "while")]
+//
+/// Loop while a condition is upheld.
+///
+/// A `while` expression is used for predicate loops. The `while` expression runs the conditional
+/// expression before running the loop body, then runs the loop body if the conditional
+/// expression evaluates to `true`, or exits the loop otherwise.
+///
+/// ```rust
+/// let mut counter = 0;
+///
+/// while i < 10 {
```

Comment:
```suggestion
/// while counter < 10 {
```
Shouldn't it be `counter`? There is no `i` in this scope/example, or am I mistaken?

Reply:
Yes, this should be `counter`. Thanks!